src/core/link-to-dfn.js

```diff
@@ -7,6 +7,7 @@ export const name = "core/link-to-dfn";
 export function run(conf, doc, cb) {
   doc.normalize();
   var titles = {};
+  var listOfDuplicateDfns = [];
```

Comment:
move this down to line 25, then just let the garbage collector remove it after each run of "forEach"

Reply:
additional nit, make this a "const"

Reply:
on it

Reply:
Line number 25 is this `if (oldIsDfn && newIsDfn) {`.


 Is this the appropriate place as I think in every run of `for each` for `dfn` the array would be refreshed. Rather this should happen in every run of `for each` for `title`

Reply:
Moreover making it a `const` doesn't work. As it doesn't allow any modifications to the array for pushing other `duplicate definitions` of other `tittles`

Reply:
That's odd. ok, leave it as `var` for now and I'll have a look.